src/ray/protobuf/extra_actions_base.proto

```diff
@@ -0,0 +1,184 @@
+// Copyright 2014 The Bazel Authors. All rights reserved.
```

Comment:
Does this have to be in our `src`?

Reply:
Until this issue is resolved: https://github.com/mehrdadn/ray/blob/include-what-you-use/BUILD.bazel#L846

Reply:
Not sure about Mac, but the script does run on Windows and Linux.  
The CI only runs for Linux though; I think that's enough to catch most issues.

Reply:
Ok however can we move it to thirdparty, patches, or other directory so we don't confuse this proto with ray's protos?

Reply:
Sure!